dbt_copilot_helper/commands/application.py

```diff
@@ -0,0 +1,264 @@
+def stats_long(env, app, project_profile, storage, network):
+    """Deploy image tag to a service, defaults to image tagged latest."""
+
+    # Query string to get the required container stats
+    query_string = "stats max(CpuUtilized), max(MemoryUtilized) by TaskId, ContainerName, TaskDefinitionFamily, TaskDefinitionRevision, Image, StorageReadBytes, StorageWriteBytes, NetworkRxPackets, NetworkTxBytes | filter Type='Container' | sort TaskId, ContainerName desc"
+    cpu_response = get_query_results(env, app, project_profile, query_string)
+
+    click.echo(
+        click.style(
+            f"\n{'Name:':<20}",
+            fg="green",
+        )
+        + click.style(f"{app}", fg="green", bold=True)
+    )
+
+    click.echo(
+        click.style(
+            f"{'Environment:':<20}",
+            fg="green",
+        )
+        + click.style(f"{env}", fg="green", bold=True)
+    )
+
+    click.echo(
+        click.style(
+            f"{'No of instances:':<20}",
+            fg="green",
+        )
+        + click.style(len(cpu_response["results"]), fg="green", bold=True)
+    )
+
+    index = 0
+
+    for field in cpu_response["results"]:
+        # Not all containers have an image not present, so need to pad.
+        if len(field) == 10:
+            field.insert(4, {"field": "Image", "value": ""})
+
+        task = field[0]["value"]
+        cont_name = field[1]["value"]
+        cont_name_short = field[2]["value"].split("-")[-1]
+        task_def_revision = field[3]["value"]
+        image = field[4]["value"]
+        storage_read = field[5]["value"]
+        storage_write = field[6]["value"]
+        network_read = field[7]["value"]
+        network_write = field[8]["value"]
+        cpu = "%.1f%%" % float(field[9]["value"])
+        memory = f"{field[10]['value']}M"
+
+        # Nothing to compare to at start.
+        if index > 0:
+            end_index = index - 1
+        else:
+            end_index = 0
+
+        # If its a new task then display the headings.
+        if (
+            index == 0
+            or cpu_response["results"][index][0]["value"]
+            != cpu_response["results"][end_index][0]["value"]
+        ):
+            click.echo(
+                click.style(
+                    f"\n{'Type:':<10}",
+                    fg="green",
+                )
+                + click.style(f"{cont_name_short}", fg="green", bold=True)
+            )
+
+            click.secho(
+                f"{'Task ID:':<10}{task}",
+                fg="green",
+            )
+            heading = f"{'Container Name':<35}{'CPU':<10}{'Memory':<10}{'Revision':<12}"
+
+            # Optional parameters.
+            if storage:
+                heading += f"{'Dsk Read':<12}{'Dsk Write':<12}"
+            if network:
+                heading += f"{'Net Read':<12}{'Net Write':<12}"
+            heading += "Image"
+
+            click.secho(
+                heading,
+                fg="cyan",
+            )
```

Comment:
I'm surprised Black hasn't single lined all these. Have you got the `pre-commit` stuff running?

Reply:
Gah, its black doing this.  I updated it to single line and Black comes along and splits it to multiline.

Reply:
Oh, I think it does that if you have a trailing comma on the last argument.

Reply:
spot on, I have updated this.  Now its behaving.